<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPanel</class>
 <widget class="QWidget" name="LoginPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>511</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ChatQQ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="LoginWidget" native="true">
     <widget class="ImageLabel" name="imgLB_logo">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>247</width>
        <height>141</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>247</width>
        <height>161</height>
       </size>
      </property>
     </widget>
     <widget class="HyperlinkLabel" name="lb_register">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>459</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Register for an account</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_login">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>232</y>
        <width>291</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>20</number>
      </property>
      <property name="midLineWidth">
       <number>23</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="PushButton" name="bt_login">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>128</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
       <widget class="EditableComboBox" name="comb_username">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Username</string>
        </property>
       </widget>
       <widget class="PasswordLineEdit" name="PasswordLineEdit">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
       <widget class="CheckBox" name="cb_autoLogin">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Auto Login</string>
        </property>
       </widget>
       <widget class="CheckBox" name="cb_rememberPassword">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>120</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Remember Pass</string>
        </property>
       </widget>
       <widget class="PushButton" name="quick_signin_button">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>160</y>
          <width>128</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Qucik SIgn In</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="IndeterminateProgressRing" name="lb_loading">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="CaptionLabel" name="lb_findPassword">
      <property name="geometry">
       <rect>
        <x>241</x>
        <y>459</y>
        <width>88</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>48</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Forgot Password</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>460</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout_QRcode">
       <item>
        <widget class="ToolButton" name="bt_QRcode"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HyperlinkLabel</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>IndeterminateProgressRing</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PasswordLineEdit</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>EditableComboBox</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
